App.js

```diff
@@ -6,33 +6,90 @@ import { createStackNavigator } from '@react-navigation/stack';
 import FlyBox from './containers/FlyBox';
 import AddFly from './containers/AddFly';
 import EditFly from './containers/EditFly';
+import { Ionicons, MaterialCommunityIcons } from '@expo/vector-icons';
```

Comment:
I started with Ionicons then ended up using MaterialCommunityIcons since they had a hook and I believe they have a fish as well we can use. We will want to remove this unnecessary Ionicons import if we decided not to use it.